lua/check_incr_limit.lua

```diff
@@ -1,21 +1,25 @@
 -- Credit: http://www.dr-josiah.com/2014/11/introduction-to-rate-limiting-with_26.html
--- there is enough resources, update the counts
-for i, limit in ipairs(limits) do
-    local saved = saved_keys[i]
 
-    for j, key in ipairs(KEYS) do
-        -- update the current timestamp, count, and bucket count
-        redis.call('HSET', key, saved.ts_key, saved.trim_before)
-        redis.call('HINCRBY', key, saved.count_key, weight)
-        redis.call('HINCRBY', key, saved.count_key .. saved.block_id, weight)
+-- Return if the key(s) are whitelisted or blacklisted
+if not is_whitelisted and not is_blacklisted then
```

Comment:
This is the main diff in this file, the rest of the diff is due to changing indent levels.